packages/web3-common/src/formatters.ts

```diff
@@ -260,7 +263,7 @@ export const outputTransactionFormatter = (tx: TransactionInput): TransactionOut
  * @param {Object} log object
  * @returns {Object} log
  */
-export const outputLogFormatter = (log: LogsInput): LogsOutput => {
+export const outputLogFormatter = (log: Partial<LogsInput>): LogsOutput => {
```

Comment:
Ideally these formatters would be replaced by the same process we're using for formatting `web3-eth` responses. This could probably exist as a cleanup `TODO`

Reply:
Yes. It would be.